search-sentiments-map/src/main/webapp/script.js

```diff
@@ -12,9 +12,138 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** Creates the world map. */
-function createWorldMap() {
-  map = new google.maps.Map(
-      document.getElementById('map'),
-      {center: {lat: 38.46049, lng: -5.428423}, zoom: 3});
+let mapStyle = [{
+  'stylers': [{'visibility': 'off'}],
+}, {
+  'featureType': 'landscape',
+  'elementType': 'geometry',
+  'stylers': [{'visibility': 'on'}, {'color': '#fcfcfc'}],
+}, {
+  'featureType': 'water',
+  'elementType': 'geometry',
+  'stylers': [{'visibility': 'on'}, {'color': '#bfd4ff'}],
+}];
+let map;
+let dataMin = Number.MAX_VALUE;
+let dataMax = -Number.MAX_VALUE;
+let infowindow;
+
+
+/** Loads the map with country polygons when page loads. */
+function initMap() {
+  map = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 29.246630, lng: 29.678410},
+    zoom: 3,
+    styles: mapStyle
+  });
+
+  infowindow = new google.maps.InfoWindow({});
+
+  // Set up the style rules and events for google.maps.Data.
+  map.data.setStyle(styleFeature);
+  map.data.addListener('mouseover', mouseInToRegion);
+  map.data.addListener('mouseout', mouseOutOfRegion);
+
+  loadMapShapes();
+}
+
+/** Loads the country boundary polygons from a GeoJSON source. */
+function loadMapShapes() {
+  // Load country outline polygons from a GeoJson file.
+  map.data.loadGeoJson('countries.geojson');
+
+  // Load country data whenever a new feature(country) is added.
+  google.maps.event.addListener(map.data, 'addfeature', function() {
+    loadCountryData();
+  });
+}
+
+/** Loads the country data by randomly generating an int below 100. */
+function loadCountryData() {
```

Comment:
Maybe loadCountrySentimentData?

Reply:
Not sure, because I would think in the future this may include more than just the sentiment data, but possible the other data as well.

Reply:
I think we'll have different functions for different data, but yeah we can keep it as this for now